doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -2035,32 +2033,49 @@ If the Suricata deployment is using multiple NICs on different NUMA nodes then h
 DPDK memory pools hold packets received from NICs. These memory pools are allocated in hugepages.
 One memory pool is allocated per interface. The size of each memory pool can be individual and is set with
 the `mempool-size`. Memory (in bytes) for one memory pool is calculated as: `mempool-size` * `mtu`.
-Sum of memory pool requirements divided by the size of one hugepage results in the number of required hugepages.
-It causes no problem to allocate more memory than required but it is vital for Suricata to not run out of hugepages.
+The sum of memory pool requirements divided by the size of one hugepage results in the number of required hugepages.
+It causes no problem to allocate more memory than required, but it is vital for Suricata to not run out of hugepages.
 
-Mempool cache is local to the individual CPU cores and holds packets that were recently processed. As the mempool is
-shared among all cores, cache tries to minimize the required inter-process synchronization. Recommended size of the cache
+The mempool cache is local to the individual CPU cores and holds packets that were recently processed. As the mempool is
+shared among all cores, the cache tries to minimize the required inter-process synchronization. The recommended size of the cache
 is covered in the YAML file.
 
-There has been an ongoing effort to add a DPDK support into Suricata. While the capture interface is continually evolving,
-there has been certain areas with an increased focus. The current version of the DPDK capture interface provides
-support for physical NICs and for running on physical machines in workers runmode.
-The work has not been tested neither with the virtual interfaces nor
-in the virtual environments like VMs, Docker or similar.
+To be able to run DPDK on Intel cards, it is required to change the default Intel driver to either
+`vfio-pci` or `igb_uio` driver. The process is described in
+`DPDK manual page regarding Linux drivers <https://doc.dpdk.org/guides/linux_gsg/linux_drivers.html>`_.
+DPDK is natively supported by Mellanox and thus their NICs should work "out of the box".
 
-Although the capture interface uses DPDK library, there is no need to configure any lcores.
-The capture interface uses the standard Suricata threading module.
-Additionally, Suricata is intended to run as a primary process only.
+**Current DPDK support** involves Suricata running on:
+
+* a physical machine with a physical NICs such as:
+    * mlx5 (ConnectX-4/ConnectX-5/ConnectX-6)
+        * MCX416A-CCAT
```

Comment:
what is the purpose of listing these precise nics? think mentioning the higher level products (cx4/5/6, x(l)710, e810) is enough

Reply:
Seems like something worth documenting, but not in a user guide but rather in something like release notes